crates/bevy_ui/src/layout/mod.rs

```diff
@@ -97,20 +114,20 @@ pub fn ui_layout_system(
     mut node_transform_query: Query<(&mut Node, &mut Transform)>,
     #[cfg(feature = "bevy_text")] mut text_pipeline: ResMut<TextPipeline>,
 ) {
-    struct CameraLayoutInfo {
-        size: UVec2,
-        resized: bool,
-        scale_factor: f32,
-        root_nodes: Vec<Entity>,
-    }
+    let UiLayoutSystemBuffers {
+        interned_root_notes,
+        resized_windows,
+        camera_layout_info,
+    } = &mut *buffers;
 
     let default_camera = default_ui_camera.get();
     let camera_with_default = |target_camera: Option<&TargetCamera>| {
         target_camera.map(TargetCamera::entity).or(default_camera)
     };
 
-    let resized_windows: HashSet<Entity> = resize_events.read().map(|event| event.window).collect();
-    let calculate_camera_layout_info = |camera: &Camera| {
+    resized_windows.clear();
```

Comment:
This doesn't really matter since we keep allocations, but shouldn't we clear at the end?